infrastructure/template.yaml

```diff
@@ -1417,9 +1061,11 @@ Resources:
     Condition: "DeployAlarms"
     Properties:
       OKActions:
-        - !ImportValue platform-alarm-warning-alert-topic
+        - !ImportValue core-infrastructure-AlarmTopic
```

Comment:
If we are thinking about alarms that go to pager duty as things that are so critical we should wake someone up in the middle of the night if they're broken, I don't think abandoning really falls into that category. What do you think?

Reply:
Oh also I see it's not in scope on the ticket

Reply:
That makes sense as it doesn't stop the overall flow of the users journey. I can revert back and leave it as the warning topic then